addons/product/tests/test_variants.py

```diff
@@ -1230,3 +1230,154 @@ def write(self, vals):
             product.write({'name': 'Bar', 'description': 'Bar'})
             self.assertEqual(product.name, 'Bar')
             self.assertEqual(product.description, 'Bar')
+
+
+class TestVariantsExclusion(common.TestProductCommon):
+
+    def setUp(self):
+
```

Comment:
overkill empty lines a bit everywhere in this file

Reply:
Side question, do we really need a setup that long in this new test class to cover the indented use cases?
(note: I didn't check the new tests in details)